lua/telescope/_extensions/neorg/insert_file_link.lua

```diff
@@ -74,15 +74,18 @@ local function generate_links(preview)
             if preview then
                 title = get_file_title(file)
                 if title then
-                    title_display = " [" .. title .. "]"
+                    title_display = "| " .. title
                 end
             end
 
             file = Path(file)
             local relative = file:relative_to(Path(files[1]))
+            local relative_string = relative:remove_suffix(".norg"):tostring()
+
+            local padding = 100 - #relative_string
             local links = {
                 file = file,
-                display = "$/" .. relative .. title_display,
+                display = "󰈙 " .. relative_string .. string.rep(" ", padding) .. title_display,
```

Comment:
this change just makes things look ugly with that vertical line floating around somewhere

Reply:
talking about the padding between the path and the title display

Reply:
Oh ok, my intention was to add some separation to make the titles easier to read, but I clearly missed the mark.